rest/route/service.go

```diff
@@ -88,6 +88,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/distros/{distro_id}/icecream_config").Version(2).Patch().Wrap(editHosts).RouteHandler(makeDistroIcecreamConfig(sc, env))
 
 	app.AddRoute("/hooks/github").Version(2).Post().RouteHandler(makeGithubHooksRoute(sc, opts.APIQueue, opts.GithubSecret, settings))
+	app.AddRoute("/hooks/sns").Version(2).Post().RouteHandler(makeAwsSnsRoute(sc))
```

Comment:
I think this should be /hooks/aws instead of sns, (or at least aws/sns) since this is an AWS listener, that happens to listen to sns (kind of like how above the hook is for github, which is the actual service, not the name of how github sends updates, although I don't know if github has a name for it)

Reply:
done